boards/st/stm32h573i_dk/stm32h573i_dk.dts

```diff
@@ -273,9 +273,10 @@
 
 	status = "okay";
 
-	mx25lm51245: ospi-nor-flash@90000000 {
+	mx25lm51245: ospi-nor-flash@0 {
 		compatible = "st,stm32-xspi-nor";
-		reg = <0x90000000 DT_SIZE_M(64)>; /* 512 Mbits */
+		reg = <0>;
+		size = <0x20000000>; /* 512 Mbits */
```

Comment:
Nit: use `DT_SIZE_K/M()` macros everywhere possible to ease reading.

Reply:
done